src/safeds/ml/nn/_forward_layer.py

```diff
@@ -24,11 +26,13 @@ def __init__(self, input_size: int, output_size: int, activation_function: str):
                     self._fn = nn.ReLU()
                 case "softmax":
                     self._fn = nn.Softmax()
+                case "none":
```

Comment:
Should this be allowed? What is a use case for not having an activation function?

Reply:
Yes, a multi-class classification uses the `CrossEntropyLoss`. This loss does already apply a `Softmax` function. Therefore, the last layer of a multi-class Classification should not have an activation function. See #581